app/views/shoppinglists/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<section class="shoppinglist-show">
+  <p style="color: green"><%= notice %></p>
+
+  <h1>Shopping List</h1>
+
+  <div class="shoppinglist-show-headline">
+    <p>
+      Ammout of food to buy: <%= @recipefoods_count %>
+    </p>
+    <p>
+      Total value of food needed: <%= number_to_currency(@total_price) %>
+    </p>
+  </div>
+
+  <Table border="1">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @recipefoods.each do |recipefood| %>
+        <tr>
+          <td><%= link_to recipefood.food.name, user_food_path(recipefood.food.user, recipefood.food) %></td>
+          <td><%= "#{recipefood.quantity} #{recipefood.food.measurement_unit}" %></td>
+          <td><%= number_to_currency(recipefood.food.price * recipefood.quantity) %></td>
+        </tr>
+      <% end %>
+    </tbody>
```

Comment:
- [ ] You did a great job working on the changes suggested by the previous reviewer! 💯 However, according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/233489960-4b4afdda-f712-4991-8a34-023d42b2ba29.png)

The goal of this general shopping list is to list the missing food for all the recipes of the user, so for example, if the user has 5kg of `food1`, and one recipe asks for 10kg, the shopping list should show 5kg and not 10kg, which is what the user needs to buy in order to have enough for the recipe, take in mind that it should list also ingredients from other recipes if there's more than one! 🤓 🔧  

💡 Hint: Compare the `recipefoods` quantity for the user's recipes with the user's `foods` quantity.

User foods:
![image](https://user-images.githubusercontent.com/77069011/233490350-094ea408-a494-41ad-b2fd-095f154810c7.png)

Ingredients needed for the recipe:
![image](https://user-images.githubusercontent.com/77069011/233490441-ea212491-0849-4f40-b1ab-c22635d60fb1.png)

General shopping list:
![image](https://user-images.githubusercontent.com/77069011/233490675-8fa470e5-caea-40e3-902c-9b76f1028f55.png)

Please consider taking a look at this and fixing it! 🤓 🔧

Reply:
Thank you for the explanation